project/accounts/authentication.py

```diff
@@ -5,9 +5,16 @@
 from django.utils.http import urlsafe_base64_encode
 from django.utils.http import int_to_base36
 from django.template.loader import render_to_string
+from rest_framework import authentication
 from accounts.utils import send_email
 
 
+class CsrfExemptSessionAuthentication(authentication.SessionAuthentication):
```

Comment:
We should add a note to deprecate this authentication method and follow the Django/DRF CSRF guidelines.
```suggestion
# TODO: deprecate this authentication method and follow DRF CSRF guidelines
# https://www.django-rest-framework.org/topics/ajax-csrf-cors/
class CsrfExemptSessionAuthentication(authentication.SessionAuthentication):
```